### DISCUSSION TITLE
Is Axios 1.x considered stable yet? If not is Axios 0.x still going to be in support in case of vulnerability?

### DISCUSSION THREAD

**Discussion by lnjbr** [axios/axios#5645] [Q&A]
Hey there :wave: 

In attempting to upgrade to Axios 1.x without a migration guide, I feel like I've been running into complete blockers. I would be more than down to start working around those blockers if there was validation that Axios in its current state _is_ stable, but it seems like a lot of the 1.x releases have been primarily composed of fixes to functionality that was broken in the initial 1.x release. Without proper documentation on what the breaking changes are, it's hard to determine what is a bug that is in the process of being resolved, and what is just a set change of Axios 1.x that will need to be worked around regardless of future stability.

In the case that Axios 1.x is not stable, upgrading without an upgrade guide seems unnecessarily Sisyphean. I would love to wait for confirmed stability, but that also raises concerns with whether or not Axios 0.x will still be in support in the case of something like a critical vulnerability arising. 

A [comment](https://github.com/axios/axios/issues/5114#issuecomment-1277169905) in a closed issue (#5114) from October stated that Axios 1.x was _not_ stable, and that Axios 0.x _would_ continue to be in support, but the same comment stated that it would be receiving an update soon, which it hasn't, and I haven't been able to find anything on 1.x stability/0.x support since then (the last comment in the issue has been unaddressed for two weeks).

Would appreciate having this cleared up so that my team and I can determine next steps for our app. Thanks!

**Comment by bmuenzenmeyer** [comment]
👋 

Not sure - but I've quick created https://github.com/bmuenzenmeyer/axios-1.0.0-migration-guide/

- we as a community of 99k stars and 40+ million weekly downloads have the collective wisdom to track these changes, spread the word, add what you can! I'll be encouraging my team to do so.
- (no warranties, but I hope it helps some folks until something official lands)
